plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiModule.java

```diff
@@ -65,6 +66,7 @@ public void configure(Binder binder)
         configBinder(binder).bindConfig(ParquetReaderConfig.class);
         configBinder(binder).bindConfig(ParquetWriterConfig.class);
 
+        binder.bind(HdfsEnvironment.class).in(Scopes.SINGLETON);
```

Comment:
Do we really need this additional binding? `HdfsEnvironment` is already injected in the `HudiSplitManager`.